### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by jordisala1991**

### CODE REVIEW

**Comment by VincentLanglet** [comment]
rector is failing

**Comment by jordisala1991** [comment]
Yes, probably related to the 1.6.0 release of phpstan

**Comment by jordisala1991** [comment]
This release: https://github.com/phpstan/phpstan-strict-rules/releases/tag/1.2.0

makes the rector unhappy, because rector is using the 1.5.7 version of phpstan

File: composer.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
         "psalm/plugin-phpunit": "^0.16",
         "psalm/plugin-symfony": "^3.0",
         "psr/event-dispatcher": "^1.0",
-        "rector/rector": "^0.12",
+        "rector/rector": "^0.12@dev",
```
Comment (jordisala1991) [inline composer.json]:
```suggestion
        "rector/rector": "^0.12",
```

**Pull request merged**